tests/github/interactive_api_director/experiments/pytorch_kvasir_unet/envoy/envoy_config.yaml

```diff
@@ -8,4 +8,4 @@ shard_descriptor:
   params:
     data_folder: kvasir_data
     rank_worldsize: 1,90
-    enforce_image_hw: [300, 400]
\ No newline at end of file
+    enforce_image_hw: '300, 400'
```

Comment:
did you turn a list into one string? What is the reason?

Reply:
It was a bug, we use string in the current implementation.